data/autoyast_containers/autoyast_containers.xml.ep

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <addons config:type="list">
+      % if ($get_var->('VERSION') =~ /15|15-SP1|15-SP2|15-SP3/) {
+      <addon>
+        <name>sle-module-basesystem</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      <addon>
+        <name>sle-module-server-applications</name>
+        <version>{{VERSION}}</version>
+        <arch>{{ARCH}}</arch>
+      </addon>
+      % }
+      <addon>
+        <name>sle-module-containers</name>
+        % if ($get_var->('VERSION') =~ /12-SP5|12-SP4|12-SP3/) {
+        <version>12</version>
+        % } else {
+        <version>{{VERSION}}</version>
+        % }
+        <arch>{{ARCH}}</arch>
+      </addon>
+    </addons>
+  </suse_register>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <bootloader>
+    % unless ($check_var->('ARCH', 's390x')) {
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/vda</linux>
+      </device_map_entry>
+    </device_map>
+    % }
+    <global>
+      % if ($check_var->('ARCH', 's390x')) {
+      <activate>false</activate>
+      <boot_extended>false</boot_extended>
+      <boot_mbr>true</boot_mbr>
+      <boot_root>true</boot_root>
+      <generic_mbr>false</generic_mbr>
+      % } elsif ($check_var->('ARCH', 'ppc64le')) {
+      <append>showopts plymouth.ignore-serial-consoles console=hvc0 console=tty mitigations=auto crashkernel=201M</append>
+      <boot_boot>false</boot_boot>
+      % } else {
+      <append>splash=verbose</append>
+      % }
+      <timeout config:type="integer">-1</timeout>
+    </global>
+    % if ($check_var->('ARCH', 'ppc64le') or $check_var->('ARCH', 's390x')) {
+    <loader_type>grub2</loader_type>
+    % }
+  </bootloader>
+  <networking>
+    % if ($check_var->('ARCH', 's390x')) {
+    <interfaces config:type="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <device>eth0</device>
+        <dhclient_set_default_route>yes</dhclient_set_default_route>
+        <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+    % } else {
+    <keep_install_network config:type="boolean">true</keep_install_network>
+    % }
+  </networking>
+  <software>
+    <products config:type="list">
+      <product>SLES</product>
+    </products>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <packages config:type="list">
+      <package>iputils</package>
+      <package>acl</package>
+      % if ($check_var->('INSTALL_DOCKER', '1')) {
```

Comment:
I would like to ask you to make this variables a bit more generic (`DOCKER=1` `PODMAN=1`. That will give us some flexibility. for instance we can remove `CONTAINER_RUNTIME` and make use of those generic ones.

Reply:
I would prefer to keep `CONTAINER_RUNTIME` as it really defines what it's made for.

`INSTALL_DOCKER` yeah... it's maybe too generic and I didnt have a better idea.Maybe I can reuse `CONTAINER_RUNTIME` for this..

Reply:
I actually find this `INSTALL_{DOCKER,PODMAN}` self-explanatory in deploy VM scenario with certain packages and modules.

Reply:
if we want to be even more specific, what about `AUTOYAST_INSTALL_DOCKER=1`  and `AUTOYAST_INSTALL_PODMAN=1`

Reply:
Yes, that works, too. Given the context, `INSTALL_$runtine` is understandable well enough. Also, it is versatile as it can be used outside AY. :)

Reply:
`INSTALL_$runtine` is self-explanatory indeed. just my point was to use just `PODMAN` and `DOCKER` which their present in the variables can indicate clearly the scope for all the actions. However i would like to resolve this and focus in other things in the PR. feel free to reopen it if you think it needs to.